operators_02/angle_pairs.py

```diff
@@ -3,5 +3,12 @@
 # for more info on this quiz, go to this url: http://www.programmr.com/anglepairs
 
 
-def angle_pairs(a, b, c):
-    pass
+def angle_pairs(x, y, z):
+    if x + y == 90 or x + z == 90 or y + z == 90 or x + y == 180 or x + z == 180 or y + z == 180:
```

Comment:
This will still fail the test, remember we want to check for both complimentary and supplementary angles this mans that we have to use and, not or

Also this line of code can be reduced to something that looks better, one way would be;

```python
a = x + y
b = x + z
c = y + z

# and then use the vars to check if there is a 90 and 180
```

There is a simple way but complex way to check for 90 and 180 using list methods - don't do it right now, we will look at it in lists section